data/tutorial/02_command-line.md

```diff
@@ -106,13 +114,14 @@ let () =
   Irmin_cli.(run ~default commands)
 ```
```

Comment:
This isn't for the command line, right? Is it to alter a specific file, like the `irmin` executable file?